### PR TITLE
New GUI for Reown Connection Provider

### PR DESCRIPTION
**Pull request by creeppak**
## What's in the PR

This PR replaces the outdated Reown GUI with a new one provided to us by @serenaho. It also introduces some changes to the Reown module, but it's still mostly visual updates.

One new thing worth mentioning is `OperationTracker`. This is a new service which allows you notify users when there is long-lasting operation currently being executed. It should be used instead of directly requesting `InfoOverlayManager` to show loading overlay.

Wallet icons loading was also fixed.

## What should be tested

- Lansdcape and Portrait modes for the new Reown GUI.
- Desktop, iOS and Android platforms.

## Screenshots
<details>
  <summary>Unfold me</summary>

image
image

</details>

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs [hunk 1]
```diff
@@ -105,8 +103,8 @@ private void OnValidate()
     {
         if (!modalScreenFactory.LandscapePrefab && !modalScreenFactory.PortraitPrefab)
         {
-            modalScreenFactory.LandscapePrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>("Packages/io.chainsafe.web3-unity.web3auth/Runtime/Prefabs/W3AConnectionScreen_L.prefab");
-            modalScreenFactory.PortraitPrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>("Packages/io.chainsafe.web3-unity.web3auth/Runtime/Prefabs/W3AConnectionScreen_P.prefab");
+            modalScreenFactory.LandscapePrefab = UnityEditor.AssetDatabase.LoadAssetAtPath<GuiScreen>(UnityEditor.AssetDatabase.GUIDToAssetPath("9e5f859444d8b4a448e79b28a6033fd7"));
```
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs:106]: Yikes, why are we using GUID and not the path?

**Review by juans-chainsafe — CHANGES_REQUESTED** [review]
Getting build errors:
```
/Users/juanmanuelspoleti/Desktop/workspace/web3.unity/Packages/io.chainsafe.web3-unity/Runtime/Scripts/Connection/IConnectionHandler.cs(11,34): error CS0234: The type or namespace name 'Operations' does not exist in the namespace 'ChainSafe.Gaming.Web3.Core' (are you missing an assembly reference?)

/Users/juanmanuelspoleti/Desktop/workspace/web3.unity/Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/GuiOperationNotificationHandler.cs(2,34): error CS0234: The type or namespace name 'Operations' does not exist in the namespace 'ChainSafe.Gaming.Web3.Core' (are you missing an assembly reference?)

/Users/juanmanuelspoleti/Desktop/workspace/web3.unity/Packages/io.chainsafe.web3-unity/Runtime/Scripts/UI/GuiOperationNotificationHandler.cs(6,52): error CS0246: The type or namespace name 'IOperationNotificationHandler' could not be found (are you missing a using directive or an assembly reference?)
```

File: Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs (hunk 1 above)
Comment (creeppak) [reply Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs:106]: It a safety measure. When you move/rename file its path changes, but GUID always stays the same.

**Comment by creeppak** [comment]
> Getting build errors:

Automatic DLL build is broken. Build and pushed the updated DLLs manually. You can try it again.

File: Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs (hunk 1 above)
Comment (kantagara) [reply Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthConnectionProvider.cs:106]: Ah fair, those things will never change their GUID.

**Review by kantagara — APPROVED** [review]

**Review by juans-chainsafe — CHANGES_REQUESTED** [review]
Not always, but in **iOS** Im stuck with the "Switching wallet network..." message, is true that the first time it require to switch from sepolia to mainnet, but after that it keeps trying to switch something that is already switched.
IMG_1353

Also, if I close the app and open it again, it automatically tries to connect to Metamask, which is a weird behaviour. 

I see more wallets now in Desktops which is nice!

**Review by rob1997 — APPROVED** [review]

**Pull request merged**